fe/fe-core/src/main/java/com/starrocks/planner/PaimonScanNode.java

```diff
@@ -170,17 +168,16 @@ protected String getNodeExplainString(String prefix, TExplainLevel detailLevel)
             output.append("\n");
         }
 
-        output.append(prefix).append(String.format("avgRowSize=%s", avgRowSize));
-        output.append("\n");
+        output.append(prefix).append(String.format("avgRowSize=%s\n", avgRowSize));
 
-        output.append(prefix).append(String.format("numNodes=%s", numNodes));
-        output.append("\n");
+        output.append(prefix).append(String.format("numNodes=%d\n", numNodes()));
```

Comment:
this should be removed because I remove numNodes yesterday.

Reply:
ok